### PR TITLE
[How to guides] Whatsapp section

### PR DESCRIPTION
**Pull request by vasvi-sood**
whatsapp1
whatsapp2
whatsapp3

### CODE REVIEW

File: docusaurus/docs/howto/deploy-whatsapp.md [hunk 1]
```diff
@@ -0,0 +1,130 @@
+---
+title: Deploy in Whatsapp
+---
+
+Whatsapp  is the most popular IM platform, so this how to guide focusses to deploy the bot in Whatsapp locally.
```
Comment (proudofsimin) [inline docusaurus/docs/howto/deploy-whatsapp.md]: typo: focuses on deploying

**Review by abhishek-iiit — APPROVED** [review]
Approved PR with suggested changes. If you could have a little more explanation to the header section, it can become prefect PR, rest everything is perfect :)

**Review by proudofsimin — CHANGES_REQUESTED** [review]
Could you modify according to the advice I gave in #1098 ? Thank you.

**Comment by vasvi-sood** [comment]
> Could you modify according to the advice I gave in #1098 ? Thank you.

Sure

**Comment by vasvi-sood** [comment]
All changes have been made. Kindly review it @proudofsimin  and @Rohitesh-Kumar-Jain

**Review by proudofsimin — APPROVED** [review]
Thanks

**Review by Rohitesh-Kumar-Jain — APPROVED** [review]

**Review by huan — CHANGES_REQUESTED** [review]
```diff
- docusaurus/static/img/docs/howto/IM_platform/deploy-whatsapp.webp 
+ docusaurus/static/img/docs/howto/deploy-whatsapp/deploy-whatsapp.webp 
```

**Comment by vasvi-sood** [comment]
> ```diff
> - docusaurus/static/img/docs/howto/IM_platform/deploy-whatsapp.webp 
> + docusaurus/static/img/docs/howto/deploy-whatsapp/deploy-whatsapp.webp 
> ```

All necessary changes have been made. Kindly review it @huan

File: docusaurus/docs/howto/deploy-whatsapp.md [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+
+<li> Add the dependencies for using the bot with Whatsapp.</li>
+
+```bash
+npm install wechaty-puppet-whatsapp
+```
+
+<li> Create a new folder `src` and add a file `my-bot.js`. Add your custom functions to the code snippet below:</li>
+
+```bash
```
Comment (huan) [inline docusaurus/docs/howto/deploy-whatsapp.md]: Here should be `ts` instead of `bash`

File: docusaurus/docs/howto/deploy-whatsapp.md [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+const {
```
Comment (huan) [inline docusaurus/docs/howto/deploy-whatsapp.md:98]: Please change them from CommonJS `require` to ESM `import`.

See: #1275

**Review by huan — APPROVED** [review]
Thank you very much for adding this whatsapp how-to guide!

Please follow my reviews and resolve the conflicts, then I believe it's ready to go.

**Comment by Rohitesh-Kumar-Jain** [comment]
@vasvi-sood and @abhishek-iiit pls solve the merging conflict, and help yourself in merging the PR

**Pull request merged**